packages/extract-react-types/src/index.js

```diff
@@ -98,8 +98,24 @@ function convertReactComponentClass(path, context) {
 }
 
 function convertReactComponentFunction(path, context) {
-  // we have a function, assume the props are the first parameter
-  let propType = path.get('params.0.typeAnnotation');
+  const variableDeclarator = path.findParent(scopedPath => scopedPath.isVariableDeclarator());
+  // const parent = path.findParent(scopedPath => scopedPath.isVariableDeclarator());
+  let propType;
+
+  // check for a component typed with the `FC<Props>` annotation
+  if (
+    variableDeclarator &&
+    variableDeclarator.node.id &&
+    variableDeclarator.node.id.typeAnnotation &&
+    variableDeclarator.node.id.typeAnnotation.typeAnnotation.typeName.name === 'FC' &&
+    variableDeclarator.node.id.typeAnnotation.typeAnnotation.typeParameters
```

Comment:
There has got to be a better way to do this 🤔

Reply:
yeah this a bit of a weird one. since we are "extract react types" tho - any not something more generic - im inclined to give this a pass 😅

Reply:
Oh you mean because we're supporting react specific types?

Reply:
exactly